Lib/javascript/napi/javascriptruntime.swg

```diff
@@ -31,6 +31,7 @@
 #include <stdlib.h>
 #include <assert.h>
 #include <map>
+#include <vector>
```

Comment:
Please don't add STL types as a mandatory dependency for all users. `<map>` should not be in there too! We keep the dependencies imposed on users to an absolute minimum.